Rakefile

```diff
@@ -162,16 +274,7 @@ namespace :plugin do
       sh "git", "clean", "-dfx", "unity"
       # remove cocoa build area
       FileUtils.rm_rf cocoa_build_dir
-      unless is_windows?
```

Comment:
Is this left from trying to fix android builds?

Reply:
Yes - I've reverted the change now. Thanks for spotting.